### PR TITLE
[PCX-2792] Fix expiration date format and color

### PR DESCRIPTION
**Pull request by caiomello**
| Before | After |
| - | - |
|  |  |

### CODE REVIEW

File: Sources/PayoneerCheckout/UI/Core/ExpirationDateFormatter.swift [hunk 1]
```diff
@@ -17,7 +17,7 @@ struct ExpirationDateFormatter {
     var text: String {
         get throws {
             guard let month = month, let year = year else { throw ExpirationDateFormatterError.invalidInput }
-            return "\(month)" + " / " + String(year).suffix(2)
+            return "\(String(format: "%02d", month))" + " / " + String(year).suffix(2)
```
Comment (Sorix) [inline Sources/PayoneerCheckout/UI/Core/ExpirationDateFormatter.swift:20]:
```suggestion
            return (String(format: "%02d", month) + " / " + String(year).suffix(2)
```

File: Sources/PayoneerCheckout/UI/Scenes/List/Table/DataSource/List.Table.DataSource.GroupedNetworkRow.swift [hunk 2]
```diff
@@ -73,7 +68,7 @@ extension List.Table.DataSource.GroupedNetworkRow {
         let cell = tableView.dequeueReusableCell(List.Table.DetailedLabelCell.self, for: indexPath)
 
         // TODO: We agreed to remove multiple images support so that's why we use the first image (it's card's icon). Better to updated model later to fully remove multiple logos support.
-        cell.configure(logo: logoImages.first, title: primaryLabel, subtitle: secondaryLabel)
```
Comment (Sorix) [inline Sources/PayoneerCheckout/UI/Scenes/List/Table/DataSource/List.Table.DataSource.GroupedNetworkRow.swift:76]: Could you explain what did change here? Is it not used anymore?

Comment (Sorix) [reply Sources/PayoneerCheckout/UI/Scenes/List/Table/DataSource/List.Table.DataSource.GroupedNetworkRow.swift:76]: [Resolved](https://payoneer-mrs.slack.com/archives/D02B76VLZE1/p1647853926278349)

**Review by Sorix — DISMISSED** [review]

File: Sources/PayoneerCheckout/UI/Core/ExpirationDateFormatter.swift (hunk 1 above)
Comment (caiomello) [reply Sources/PayoneerCheckout/UI/Core/ExpirationDateFormatter.swift]: Nice catch. (No need for the parenthesis so I did it in a separate commit)

**Review by siebeprojects — APPROVED** [review]

**Review by Sorix — APPROVED** [review]

**Pull request merged**